app/models/physical_ticket.rb

```diff
@@ -0,0 +1,6 @@
+class PhysicalTicket < ActiveRecord::Base
+  belongs_to :ticket_purchase
+  delegate :ticket, to: :ticket_purchase
```

Comment:
`delegate` is for expose contained objects’ public methods as your own. Ref: https://apidock.com/rails/Module/delegate

for this you should use `:through` associations. See http://guides.rubyonrails.org/association_basics.html#the-has-one-through-association